CONTRIBUTING.md

```diff
@@ -81,6 +81,10 @@ Examples:
     Qt: Add feed bump button
     Trivial: Fix typo in init.cpp
 
+Note that translations should not be submitted as pull requests, please see
+[Translation Process](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md) 
```

Comment:
would prefer 2410630 to be sqashed into bbb53b3d49e3def4b2e7c0147e46e96387e9996c